Cargo.toml

```
@@ -38,12 +48,15 @@ libc = "0.2.154"
 phf = "0.11.2"
 phf_codegen = "0.11.2"
 textwrap = { version = "0.16.1", features = ["terminal_size"] }
+thiserror = "1.0.63"
 xattr = "1.3.1"
 tempfile = "3.10.1"
 rand = { version = "0.8.5", features = ["small_rng"] }
 bytesize = "1.3.0"
 chrono = { version = "0.4.38", default-features = false, features = ["clock"] }
 walkdir = "2.5.0"
+prettytable-rs = "0.10.0"
+nix = { version = "0.28", default-features = false }
```

I would use the most recent version:
```suggestion
nix = { version = "0.29", default-features = false }
```